cms/toolbar/items.py

```diff
@@ -286,7 +286,7 @@ class FrameItem(BaseItem):
     def __init__(self, name, url, active=False, disabled=False,
                  extra_classes=None, on_close=None, side=LEFT):
         super(FrameItem, self).__init__(side)
-        self.name = "%s ..." % force_text(name)
+        self.name = "%s..." % force_text(name)
```

Comment:
Any specific reason to do this? This may breaks tests for the toolbar in third party applications